### ISSUE TITLE
Should Identity and Actions be in separate repos?

### ISSUE THREAD

**Issue by eladalon1983** [w3c/mediacapture-handle#38]
It looks like all open issues fall squarely into either the Actions or Identity categories. I've gone ahead and labelled them as much, to make it easier for people to find the issues they care about. But the question should be asked again - should these two documents really share a repo? What are the benefits?

**Comment by eladalon1983** [comment]
> This is the compromise we reached.

Indeed, it was the compromise. But it was quite explicitly non-final. It was explicit that the future could see the documents merged (your wish), the repo separated (my wish), or the status quo maintained. It is natural that I wish to convince the group, you included, that we should move in my preferred direction. (Note that this issue is not presented as a blocker for anything.)

> I can open a separate issue suggesting Identity and Actions be in a single document

If you still believe this is desirable, then you should! (And I hope you will likewise see that issue as non-blocking.)

> I believe we already prefer discussing new labels before introducing them

I did not expect these to be controversial. I want us to employ these labels because they help prevent confusion. For example, if I file an issue about Actions, I don't need to worry that someone might misunderstand that issue as lack of consensus about Identity. And vice versa. I believe clarity is always desirable.

What's your reason opposing labelling?

**Comment by jan-ivar** [comment]
I'm saying:
* we don't need both labels and a `[]` naming scheme, so let's get rid of one (e.g. the naming scheme).
* please discuss with your co-editor before making sweeping changes, to avoid noise

I'd like to remove the naming scheme asap, to avoid confusion about whether people need to follow it when filing new issues. If we can't figure out from the issue which APIs it applies to, there's a chance it applies to both, even if the person filing it may be unaware of it.

This already happened with https://github.com/w3c/mediacapture-handle/issues/10 and https://github.com/w3c/mediacapture-handle/issues/11 (if we entertain changing configured actions)

This seems especially true for feature enhancements, or complaints over a feature falling short of a use case need.

**Comment by jan-ivar** [comment]
I agree the labels are useful to track outstanding issues for the sake of progressing the docs in the w3c process.

**Comment by eladalon1983** [comment]
> I'm saying:
> 
> * we don't need both labels and a `[]` naming scheme, so let's get rid of one (e.g. the naming scheme).
> * please discuss with your co-editor before making sweeping changes, to avoid noise

Sure. You seem to prefer labels, but not [prefixes]. Let's go ahead with those. Please either remove these yourselves, or give me your blessing to do so myself.

**Comment by eladalon1983** [comment]
Circling back here, I see you've applied a thumbs-up emoji to my previous message. FYI that these do not produce an email notification and are therefore hard to discover. They make up for it by being easy to misinterpret. :-) Would you like me to remove the [prefixes], @jan-ivar?

**Comment by eladalon1983** [comment]
Capture Handle Identity has shipped in Chrome (102) and is seeing several millions of uses per week. Meanwhile, the specification of Actions has not progressed. I am genuinely interested in seeing Actions move forward, and fast. I propose that we consider again the separation of Actions into its own repo, where there will be a clear separation of issues (e.g. issue #64), and where less time would be spent on integrating Actions prose with Identity prose. Wdyt?
